python/lsst/sims/maf/slicers/healpixSlicer.py

```diff
@@ -20,13 +20,13 @@ class HealpixSlicer(BaseSpatialSlicer):
     """Healpix spatial slicer."""
     def __init__(self, nside=128, lonCol ='fieldRA' , latCol='fieldDec', verbose=True,
                  useCache=True, radius=1.75, leafsize=100,
-                 useCamera=False, rotSkyPosColName='rotSkyPos', mjdColName='expMJD'):
+                 useCamera=False, rotSkyPosColName='rotSkyPos', mjdColName='expMJD', **kwargs):
```

Comment:
Is this kwarg just for the 'useChips' option? If so, can we figure out a consistent rule for putting items in kwargs vs explicitly specifying them? (useCamera is explicitly stated, but useChips is not ... if both are specified with reasonable values in the "next level up" then why treat them differently here?)